### PR TITLE
chore: update scripts for initializing evm rollapp

### PR DESCRIPTION
**Pull request by artemijspavlovs**
chore: add scripts to generate denom metadata, genesis accounts and update the genesis file
docs: README, add steps for denommetadata
docs: README, add steps for adding genesis accounts

## Description

----

Closes #XXX

**All** items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow-up issues.

PR review checkboxes:

I have...

- [ ]  Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ]  Targeted PR against the correct branch
- [ ]  included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Linked to the GitHub issue with discussion and accepted design
- [ ]  Targets only one GitHub issue
- [ ]  Wrote unit and integration tests
- [ ]  Wrote relevant migration scripts if necessary
- [ ]  All CI checks have passed
- [ ]  Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code)
- [ ]  Updated the scripts for local run, e.g genesis_config_commands.sh if the PR changes parameters
- [ ]  Add an issue in the [e2e-tests repo](https://github.com/dymensionxyz/e2e-tests) if necessary

SDK Checklist
- [ ] Import/Export Genesis
- [ ] Registered Invariants
- [ ] Registered Events
- [ ] Updated openapi.yaml
- [ ] No usage of go `map`
- [ ] No usage of `time.Now()`
- [ ] Used fixed point arithmetic and not float arithmetic
- [ ] Avoid panicking in Begin/End block as much as possible
- [ ] No unexpected math Overflow
- [ ] Used `sendCoin` and not `SendCoins`
- [ ] Out-of-block compute is bounded
- [ ] No serialized ID at the end of store keys
- [ ] UInt to byte conversion should use BigEndian

Full security checklist [here](https://www.faulttolerant.xyz/2024-01-16-cosmos-security-1/)

----;

For Reviewer:

- [ ]  Confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Reviewers assigned
- [ ]  Confirmed all author checklist items have been addressed

---;

After reviewer approval:

- [ ]  In case the PR targets the main branch, PR should not be squash merge in order to keep meaningful git history.
- [ ]  In case the PR targets a release branch, PR must be rebased.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,15 +82,35 @@ dymd keys add sequencer --keyring-dir ~/.rollapp_evm/sequencer_keys --keyring-ba
 SEQUENCER_ADDR=`dymd keys show sequencer --address --keyring-backend test --keyring-dir ~/.rollapp_evm/sequencer_keys`
 ```
 
-fund the sequencer account
+fund the sequencer account (if you're using a remote hub node, you must fund the sequencer account or you must have an account with enough funds in your keyring)
+
+```shell
+dymd tx bank send local-user $SEQUENCER_ADDR 1000dym --keyring-backend test --broadcast-mode block --fees 1dym -b block -y --node ${HUB_RPC_URL}
```
Comment (mtsitrin) [inline README.md:88]: redundent `--broadcast-mode block`

File: README.md
Comment (mtsitrin) [inline README.md:1]: I don't think that all the "advanced" features, and those aimed for permission deployment should be part of the main readme.

IMO all the denom metadata, genesis accounts, vesting , should be at separate md file

File: scripts/settlement/add_genesis_accounts.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+fi
+
+dymd keys add alice-genesis --keyring-backend test
+dymd keys add bob-genesis --keyring-backend test
+
+tee "$ROLLAPP_SETTLEMENT_INIT_DIR_PATH/genesis_accounts.json" >/dev/null <<EOF
+[
+  {"amount":
+      {"amount":"10000000000000000000000","denom":"a${DENOM}"},
+      "address": "$(dymd keys show bob-genesis --keyring-backend test --output json | jq -r .address)"
```
Comment (mtsitrin) [inline scripts/settlement/add_genesis_accounts.sh:17]: u can do `keys show -a` for address

File: scripts/settlement/register_sequencer_to_hub.sh [hunk 3]
```diff
@@ -4,14 +4,17 @@ EXECUTABLE="rollapp-evm"
 KEYRING_PATH="$HOME/.rollapp_evm/sequencer_keys"
 KEY_NAME_SEQUENCER="sequencer"
 
-
 #Register Sequencer
-DESCRIPTION="{\"Moniker\":\"myrollapp-sequencer\",\"Identity\":\"\",\"Website\":\"\",\"SecurityContact\":\"\",\"Details\":\"\"}";
+DESCRIPTION="{\"Moniker\":\"${ROLLAPP_CHAIN_ID}-sequencer\",\"Identity\":\"\",\"Website\":\"\",\"SecurityContact\":\"\",\"Details\":\"\"}"
 SEQ_PUB_KEY="$($EXECUTABLE dymint show-sequencer)"
+BOND_AMOUNT="100000dym"
```
Comment (mtsitrin) [inline scripts/settlement/register_sequencer_to_hub.sh:10]: it's too high bond, more that u fund the sequencer in the readme.
u can use the default gensis value for `min_bond`

**Review by mtsitrin — CHANGES_REQUESTED** [review]

File: README.md
Comment (artemijspavlovs) [reply README.md:1]: I aggree, will create a separate README and revert the original one.

**Review by omritoptix — DISMISSED** [review]

**Pull request merged**